src/renderer/components/Map.js

```diff
@@ -0,0 +1,86 @@
+const React = require('react')
+const { ipcRenderer } = require('electron')
+const mapboxgl = require('mapbox-gl')
+const geojsonExtent = require('@mapbox/geojson-extent')
+const moment = require('moment')
+const MapLayerFactory = require('./helpers/MapLayerFactory')
+
+class Map extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      points: []
+    }
+    this.renderLayer = this.renderLayer.bind(this)
+    this.onMapClick = this.onMapClick.bind(this)
+  }
+
+  componentDidMount () {
+    this.componentDidMount = Date.now()
+    console.log('componentDidMount', this.componentDidMount)
+    mapboxgl.accessToken = MapLayerFactory.getAccessToken()
+    this.map = new mapboxgl.Map(
+      {
+        container: 'map',
+        style: 'mapbox://styles/mapbox/streets-v11',
+        zoom: 12,
+        attributionControl: false
+      })
+
+    this.map.on('load', this.renderLayer)
+    this.map.on('click', this.onMapClick)
+  }
+
+  renderLayer () {
+    const { selectedChat } = this.props
+    const contacts = selectedChat.contacts
+    let allPoints = []
+
+    if (this.map.loaded()) {
+      console.log('map loaded', this.componentDidMount - Date.now())
```

Comment:
console.log

Reply:
I would like to keep these log statements for a while since on my machine I have a performance issue. It takes quite a while (like 5-6 seconds) until map is loaded. After that all is fast. Would like to compare that with others. console log statements are removed anyway when compiled for production, aren't they?